ShareBook/ShareBook.Repository/LoggingContext.cs

```diff
@@ -37,6 +37,11 @@ public static class LoggingContext
                 var original = emptyJson;
                 var updated = JsonConvert.SerializeObject(item.CurrentValues.Properties.ToDictionary(pn => pn.Name, pn => item.CurrentValues[pn]));
 
+                if (item.State == EntityState.Added)
+                {
+                    item.Property("CreationDate").CurrentValue = DateTime.UtcNow;
+                }
+
                 if (item.State == EntityState.Modified)
```

Comment:
Coloque aqui como else if. Quando for Added, ele nunca será Modified. Então não tem porque fazer essa verificação. Com else if, caso ele aceite a condição de if, ele nem vai validar os else if's.